static/js/theme-map/VerticalFullPageMapOrchestrator.js

```diff
@@ -415,6 +415,14 @@ class VerticalFullPageMapOrchestrator extends ANSWERS.Component {
    * the visible area.
    */
   searchThisArea() {
+    const numConcurrentSearchThisAreaCalls =
+      this.core.storage.get(StorageKeys.LOCATOR_NUM_CONCURRENT_SEARCH_THIS_AREA_CALLS);
+    const updatedNumSearchThisAreaCalls = numConcurrentSearchThisAreaCalls + 1 || 1;
+    this.core.storage.set(
+      StorageKeys.LOCATOR_NUM_CONCURRENT_SEARCH_THIS_AREA_CALLS,
```

Comment:
From our conversation yesterday: It is unfortunate that the only way, currently, to communicate that the trigger of a search (to all of the people listening to a search) is through this asynchronous storage. However, we do not have a way to go around this. Just a note.